executors/cmdexec/cmdexec.go

```diff
@@ -23,6 +23,7 @@ import (
 var (
 	logger     = logging.NewLogger("[cmdexec]", logging.LEVEL_DEBUG)
 	preallocRe = regexp.MustCompile("^[a-zA-Z0-9-_]+$")
+	MNTOPTS string = "rw,inode64,noatime,nouuid"
```

Comment:
Please change MNTOPTS to a more standard naming style. Something like DefaultMountOpts, perhaps?